thanos/templates/store-deployment.yaml

```diff
@@ -46,16 +46,23 @@ spec:
         - "--data-dir=/var/thanos/store"
         - "--log.level={{ .Values.store.logLevel }}"
         - "--grpc-address=0.0.0.0:{{ .Values.store.grpc.port }}"
+        {{- if .Values.cluster.enabled }}
         - "--cluster.address={{ .Values.cluster.address }}:{{ .Values.cluster.port}}"
         - "--cluster.peers={{ include "thanos.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.cluster.port }}"
+        {{- end }}
         - "--objstore.config-file=/etc/config/{{ .Values.objstore.configFile }}"
+        {{- if .Values.store.extraArgs }}
```

Comment:
Please remove these extraArgs because of #908

Reply:
Thanks, but I had a confusion with the #908 PR. I removed these changes from it - it should be merged here, as this is how it was tested.